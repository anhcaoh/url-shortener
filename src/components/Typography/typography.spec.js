import React from "react";
import renderer from "react-test-renderer";
import { H1, H2, Paragraph, Block, Text, Label } from "./index";
 
describe("Typography", () => {
    it("should render snapshot", () => {
            const component = renderer.create(<>
                <H1>Heading 1</H1>
                <H2>Heading 2</H2>
                <Paragraph>Paragraph</Paragraph>
                <Block>Block Division</Block>
                <Text>Text Span</Text>
                <Label>Label</Label>
                </>);
            let tree = component.toJSON();
            expect(tree).toMatchSnapshot(); 
    });
});