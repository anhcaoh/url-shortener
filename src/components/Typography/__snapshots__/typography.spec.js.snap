// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography should render snapshot 1`] = `
Array [
  <h1>
    Heading 1
  </h1>,
  <h2>
    Heading 2
  </h2>,
  <p>
    Paragraph
  </p>,
  <div>
    Block Division
  </div>,
  <span>
    Text Span
  </span>,
  <label>
    Label
  </label>,
]
`;
